server/lib/utils/connstr-mssql.js

```diff
@@ -0,0 +1,40 @@
+import _ from "lodash";
+
+export function parseConnectionConfig(settings) {
```

Comment:
Ok then let's keep the common utils as utils and have the specific functions like this one encapsulated in their adapter